Step3_correct_background/calculate_bias.py

```diff
@@ -0,0 +1,95 @@
+import numpy as np
+import xarray as xr
+#######################################################
+# This script takes the preprocessed daily data 
+# from tropomi/GC simulation and calculates the bias 
+# between the two using the latitudinal averages and 
+# spatial/temporal smoothing. 
+# This script is based off of Lu Shen's 
+# methods/ R script.
+#######################################################
+
+# settings to adjust default smoothing windows
+# we use a +/- 15 day time window and 3x3
+# smoothing for lat/lon
+smoothing_lat_window = 3
```

Comment:
So is this a 12 x 15 degree lat/lon window? I thought we used a 20 x 20 degree window (10-degree radius), which would be 5 x 4 grid cells.